build/build.c

```diff
@@ -283,7 +284,7 @@ static rpmRC buildSpec(BTA_t buildArgs, rpmSpec spec, int what)
     return rc;
 }
 
-rpmRC rpmSpecBuild(rpmSpec spec, BTA_t buildArgs)
+int rpmSpecBuild(rpmts ts, rpmSpec spec, BTA_t buildArgs)
```

Comment:
Just noting that this is an API break. We're breaking API in some other places too in this cycle so that's okay, just saying it aloud to remind myself basically.